packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx

```diff
@@ -135,46 +146,42 @@ Tokens: #: ${catalog.totalTokens || "N/A"}
                 <div className="product-headings-heading-4">{catalog.name}</div>
               </div>
               <Separator orientation="horizontal" className="my-[10px]" />
-              <p className="mb-auto line-clamp-3 product-body-text-3-regular whitespace-pre-wrap break-words">
-                {catalog.description}
-              </p>
-              {/* Coming in v2 */}
-              {/* <div className="flex flex-wrap gap-1 mt-auto">
-                {catalog.tags && catalog.tags.length > 0 ? (
-                  <>
-                    {catalog.tags.slice(0, MAX_VISIBLE_TAGS).map((tag, index) => (
-                      <Tag key={index} variant="lightBlue">
+              <ScrollArea.Root className="flex-grow overflow-hidden">
```

Comment:
@thewbuk The scrollArea need to be noticed the height for it to correctly calculate the scroll behavior, did you test it with lots of tags to see whether this is working?